include/zck.h.in

```diff
@@ -276,7 +284,9 @@ ssize_t zck_get_chunk_comp_data(zckChunk *idx, char *dst, size_t dst_size)
 /* Find out if two chunk digests are the same */
 bool zck_compare_chunk_digest(zckChunk *a, zckChunk *b)
     __attribute__ ((warn_unused_result));
-
+/* Get associate src chunk if any */
+zckChunk *zck_get_chunk_src(zckChunk *idx)
```

Comment:
Let's rename this to `zck_get_src_chunk(zckChunk *tgt)`

Reply:
Sure